lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -286,19 +286,19 @@ struct ModuleVisitor : public BaseVisitor {
         if (!value)
           return failure();
         unsigned offset = 0;
-        auto i32 = moore::IntType::getInt(context.getContext(), 32);
+        // auto i32 = moore::IntType::getInt(context.getContext(), 32);
         for (const auto *port : llvm::reverse(multiPort->ports)) {
           if (auto *existingPort = moduleLowering->portsBySyntaxNode.lookup(
                   con->port.getSyntax()))
             port = existingPort;
           unsigned width = port->getType().getBitWidth();
-          auto index = builder.create<moore::ConstantOp>(loc, i32, offset);
+          // auto index = builder.create<moore::ConstantOp>(loc, i32, offset);
```

Comment:
No longer needed :smiley:?

Reply:
Yeah, right! Because it is used to handle the `multiport`, the number of ports must be exact. So I think we can remove these annotation codes.